jenkins/src/cgcloud/jenkins/toil_jenkins_slave.py

```diff
@@ -48,6 +48,27 @@ def _post_install_packages( self ):
         self.__install_parasol( )
         self.__patch_distutils( )
         self.__configure_gridengine( )
+        self.__createMesosCredentialsFile( )
+        self.__setEphemeralTempDir( )
+
+    @fabric_task
+    def __createMesosCredentialsFile(self):
+        # Create the credentials file and transfer ownership to mesosbox
+        sudo( 'echo toil liot > /var/lib/mesos/credentials.txt' )
+        sudo( 'chown jenkins:jenkins /var/lib/mesos/credentials.txt')
+        sudo( 'chmod 755 /var/lib/mesos/credentials.txt')
+
+    @fabric_task
+    def __setEphemeralTempDir(self):
+        """
+        This sets the value of the environment variable TMPDIR to one located on /mnt/ephemeral.
+        This way, all temp dirs created for tests will go to /mnt/ephemeral where there is a lot
+        more disk space to carry out tests.
+        """
+        sudo( 'mkdir /mnt/ephemeral/temp' )
+        sudo( 'chown jenkins:jenkins /mnt/ephemeral/temp' )
+        sudo( 'chmod 755 /mnt/ephemeral/temp' )
+        sudo( 'echo TMPDIR=/mnt/ephemeral/temp > /etc/environment')
```

Comment:
That's a good point you bring up. I actually didn't notice it till I had to do this today. I'll set it up as a bind mount in cgcloud close to where `/mnt/ephemeral` is created.